docs-src/concepts.md

```diff
@@ -173,7 +173,31 @@ handling this:
 
 * TODO
 
-### Extensibility
+### Conformance
+
+As this API aims to cover a wide set of implementations and use cases,
+it will not be possible for all implementations to support all
+features at a given time. However, we do expect that the set of
+features that are supported to be converging. To model this in the
+API, we are taking a similar approach as with
+[SIG-CONFORMANCE][sig-conformance] work on conformance
```

Comment:
Should `sig-conformance` be a link to somewhere?

Reply:
yes -- this is a broken link